core/src/main/java/org/elasticsearch/action/admin/indices/rollover/RolloverResponse.java

```diff
@@ -39,25 +39,31 @@
     private static final String DRY_RUN = "dry_run";
     private static final String ROLLED_OVER = "rolled_over";
     private static final String CONDITIONS = "conditions";
+    private static final String CREATE_INDEX_ACKNOWLEDGED = "create_index_acknowledged";
```

Comment:
we can explicitly set the the waitForShards of the index creation to NONE and use the setting of the request to wait after the alias was acked. All variants of this are not ideal, so I would go for what you find the simplest - I tend to agree that consistency is the best here and thus leaning towards what I just described.